testing/ui-tests/src/utils/metadata_test_runner.rs

```diff
@@ -100,15 +100,10 @@ impl MetadataTestRunnerCaseBuilder {
     ///
     /// The generated code will be tidied up when the `MetadataTestRunner` that
     /// this was handed out from is dropped.
-    pub fn build<M>(self, macro_metadata: M) -> String
-    where
-        M: TryInto<Metadata>,
-        M::Error: std::fmt::Debug,
-    {
-        let macro_metadata = macro_metadata.try_into().expect("can into Metadata");
+    pub fn build(self, macro_metadata: frame_metadata::RuntimeMetadataPrefixed) -> String {
         let validation_metadata = self
             .validation_metadata
-            .unwrap_or_else(|| macro_metadata.clone());
+            .unwrap_or_else(|| clone_via_encode(&macro_metadata));
```

Comment:
was this taken care of the prior `Into/From` or why is it needed?

Reply:
This is because we were previously accepting `subxt::Metadata`, which can be cloned, but now we accept `frame_metadata::RuntimeMetadataPrefixed`, which for some reason doesn't implement Clone (despite the individual versions implementing it iirc).

We can prob improve that, but until then, this is a bit of a hack to make a clone of it by encoding and then decoding again 😆